src/entities/MovieForm/ui/MovieForm.jsx

```diff
@@ -1,106 +1,165 @@
+                <Controller
+                  name="releaseDate"
+                  control={control}
+                  render={({ field }) => {
+                    return (
+                      <input
+                        {...field}
+                        className="field"
+                        type="date"
+                        id={field.name}
+                      />
+                    );
+                  }}
+                />
+              </div>
+
             <div className="block-item">
               <label htmlFor="rating">Rating</label>
-              <input
-                className="field"
-                type="text"
-                id="rating"
+              <Controller
                 name="rating"
-                value={movie.rating}
-                onChange={handleFieldChange}
+                control={control}
+                render={({ field }) => {
+                  return (
+                    <input
+                      {...field}
+                      className="field"
+                      type="number"
+                      id={field.name}
+                    />
+                  );
+                }}
               />
             </div>
+
             <div className="block-item">
               <label htmlFor="runtime">Runtime</label>
-              <input
-                className="field"
-                type="text"
-                id="runtime"
+              <Controller
                 name="runtime"
-                value={movie.runtime }
-                onChange={handleFieldChange}
+                control={control}
+                render={({ field }) => {
+                  return (
+                    <input
+                      {...field}
+                      className="field"
+                      type="text"
+                      id={field.name}
+                    />
+                  );
+                }}
               />
             </div>
           </div>
         </div>
         <div className="block-item">
           <label htmlFor="overview">Overview</label>
-          <textarea
-            className="field"
-            type="text"
-            id="overview"
+          <Controller
             name="overview"
-            value={movie.overview }
-            onChange={handleFieldChange}
-            cols="30"
-            rows="5"
+            control={control}
+            render={({ field }) => {
+              return (
+                <textarea
+                  className="field"
+                  type="text"
+                  id={field.name}
+                  cols="30"
+                  rows="5"
+                  {...field}
+                />
+              );
+            }}
           />
         </div>
       </div>
       <div className="movie-footer">
-        <button className="movie-reset-button">RESET</button>
+        <input className="movie-reset-button" onClick={()=>reset(getMovie(movie))} type="button" value={"Reset"}/>
```

Comment:
```suggestion
        <input className="movie-reset-button" onClick={()=>reset(getMovie(movie))} type="button" value="Reset"/>
```

Reply:
also why wouldn't we just use tag `button` here?

Reply:
idk, for a moment it submitted on pressing this button, but now it works

Reply:
done